packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -34,8 +34,18 @@ export async function fetchServerResponse(
     headers[NEXT_ROUTER_PREFETCH] = '1'
   }
 
+  const isNextExport = process.env.__NEXT_CONFIG_OUTPUT === 'export'
```

Comment:
IMHO, we should not re-assign `process.env.__NEXT_CONFIG_OUTPUT === 'export'` to a variable, just like we currently don't re-assign `typeof window !== 'undefined'` to `isBrowser`.

@timneutkens has expressed the concern back in my first PR to Next.js, and here I quote:

> While it might seem like this is an optimization it actually prohibits shaking these if branches by moving them into a dynamic variable, which will increase bundle size.
>
> The code is intentionally written as-is so that if(typeof window === 'undefined') is removed in server-side bundles and if(typeof window !== 'undefined') is removed from browser bundles.

https://github.com/vercel/next.js/pull/34468#issuecomment-1042881849

Reply:
Good point! Fixed in [2059734](https://github.com/vercel/next.js/pull/47022/commits/2059734b858538c1216b1eb16d27f9fbd79a1c64)